recipes/iotop/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "iotop" %}
+{% set version = "1.26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Tomas-M/iotop/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b0a334cba89249bc7cbb87cf92cf4166911bf00dd2ea8841b572fd776018c487
+
+build:
+  skip: True  # [not linux]
+  script: make && make install PREFIX=$PREFIX
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - make
+  host:
+    - ncurses
+
+test:
+  commands:
+    - $PREFIX/sbin/iotop --version
```

Comment:
I would prefer if we put this into `bin`. We don't populate `sbin` in conda-forge normally.

Reply:
Thanks, done in [af576a3](https://github.com/conda-forge/staged-recipes/pull/26529/commits/af576a3788c01e4df5607b1a59fda934959941cf)